mypy.ini

```diff
@@ -2,7 +2,6 @@
 files = src/bodywork
 warn_return_any = True
 disallow_untyped_calls=True
-disallow_untyped_defs=True
```

Comment:
Was this showing up errors in test functions for you (despite being set to scan src only)?

Reply:
Yes and it was doing my f*cking head in.

Reply:
Mine too 😆

Reply:
We that's 'that' decided then!